airbyte-integrations/connectors/source-harvest/source_harvest/streams.py

```diff
@@ -455,3 +464,4 @@ class ProjectBudget(ReportsBase):
     """
 
     report_path = "project_budget"
+    primary_key = None
```

Comment:
don't you think this is common for all ReportsBase streams?
lets just move it there